tests/pytests/test_profile.py

```diff
@@ -16,6 +16,7 @@ def testProfileSearch(env):
   conn.execute_command('hset', '2', 't', 'world')
 
   env.expect('ft.profile', 'profile', 'idx', '*', 'nocontent').error().contains('Bad command type')
+  env.expect('ft.profile', 'idx').error().contains("wrong number of arguments for 'ft.profile'")
```

Comment:
Lets make it a new test, those long test make it very hard to understand what failed (when it failed)